tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -75,13 +74,12 @@ EFX_FIR_PARAMS=passthrough,EFX_IIR_PARAMS=passthrough,\
 EFX_DRC_COMPONENT=multiband,EFX_MBDRC_PARAMS=passthrough"
 
 "sof-hda-generic\;sof-hda-efx-mbdrc-generic-2ch\;\
-HDA_CONFIG=efx,NUM_DMICS=2,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-fir-generic-2ch.bin,\
+HDA_CONFIG=efx,NUM_DMICS=2,\
 EFX_FIR_PARAMS=passthrough,EFX_IIR_PARAMS=passthrough,\
 EFX_DRC_COMPONENT=multiband,EFX_MBDRC_PARAMS=passthrough"
 
 "sof-hda-generic\;sof-hda-efx-mbdrc-generic-4ch\;\
 HDA_CONFIG=efx,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,\
-PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-efx-generic-4ch.bin,\
```

Comment:
This includes PCH DMICs. Don't we need nhlt?

Reply:
The blob is needed for DMIC.

Reply:
All my test devices have a DMIC NHLT, even UPX has, so I felt this is unnecessary. Also I'm feeling separate builds of these for MTL, LNL, ... is superfluous.

Reply:
But it is harmless to build with NHLT, and we will use the NHLT from by default. So, why not to build with NHLT in case any device does not have NHLT?

Reply:
this NHLT will only work on cAVS, it is much cleaner if there is no NHLT in tplg than to have arch specific one without spelling out that fact.